OpenTerm/Controller/ViewController.swift

```diff
@@ -339,6 +357,261 @@ extension ViewController: UIDocumentPickerDelegate {
 
 }
 
+extension ViewController: BookmarkViewControllerDelegate {
+    
+    /// The file name of the start directory bookmark. It is stored as a static
+    /// constant so we can access it when saving or loading the bookmark and
+    /// such that the settings view controller can access is as well.
+    static let bookmarkDirectory = ".bookmarks"
+    
+    
+    /// Gets the URLs of the bookmarks that were previously saved.
+    ///
+    /// - Returns: The URLs of the saved bookmarks. If something fails, the returned array will be empty.
+    func savedBookmarkURLs() -> [URL] {
+        /**
+         *  Get the document directory of the app.
+         */
+        if let dir = FileManager.default.urls(for: .documentDirectory,
+                                              in: .userDomainMask).first {
+            
+            /**
+             *  Get the directory where the bookmarks are saved.
+             */
+            let bookmarkDirectoryURL = dir.appendingPathComponent(ViewController.bookmarkDirectory,
+                                                                  isDirectory: true)
+            
+            /**
+             *  Create the bookmarks directory (if it doesn't exist)
+             */
+            do {
+                try FileManager.default.createDirectory(at: bookmarkDirectoryURL,
+                                                        withIntermediateDirectories: true,
+                                                        attributes: nil)
+                
+                /**
+                 *  Get all files that are in the bookmarks directory.
```

Comment:
nit: Probably don't need a bunch of multi-line comments here. A simple `//` will suffice.